src/api/sarifViewerApi.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ */
+
+import * as vscode from 'vscode';
+
+export interface OpenLogFileArguments {
+    /**
+     * Specifies options indicating whether to prompt the user when the
+     * SARIF log file shcmea is needs to be ugpraded to the latest one.
+     * If not specified, the user will be prompted. (Default is true).
+     */
+    readonly promptUserForUpgrade?: boolean;
```

Comment:
Would hope to be more systematic in the phrasing here. The other two start with `Indicates weather...", yet this one is different for no apparent reason.

Also, a lone "Default is true." seem most clear. IMO redundantly adding "If not specified, the user will be prompted." reduces clarity.